sdk/textanalytics/ai-text-analytics/test/pipelineOptions.spec.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+
+import { TextAnalyticsClient, TextAnalyticsApiKeyCredential } from "../src";
+import { environmentSetup } from "./utils/recordedClient";
+
+import { record, Recorder, env } from "@azure/test-utils-recorder";
+
+describe("TextAnalyticsClient Custom PipelineOptions", function() {
+  let recorder: Recorder;
+  let credential = new TextAnalyticsApiKeyCredential(env.SUBSCRIPTION_KEY);
+
+  this.timeout(10000);
+
+  beforeEach(function() {
+    recorder = record(this, environmentSetup);
+  });
+
+  afterEach(() => {
+    recorder.stop();
+  });
+
+  it("use custom user-agent string", async () => {
+    const client = new TextAnalyticsClient(env.ENDPOINT, credential, {
```

Comment:
TBH I don't think this test is worth doing live and we should just fake out a test `HTTPClient` to avoid making a network request